src/components/organism/HeaderTopNav/index.tsx

```diff
@@ -43,53 +62,82 @@ const HeaderTopNav = () => {
     }
   }, [walletConnected?.address]);
 
+  useEffect(() => {
+    if (!walletConnectOpen) {
+      setTimeout(() => setModalStep({ step: WalletConnectSteps.stepExtensions }), 1000);
```

Comment:
setTimeout can cause an issue if it does not clean the timeout. U have to clean timeout on unmount component
useEffect(() => {
    const timeoutID = setTimeout(() => {
      setIsShown(true);
    }, 1000);

    return () => {
      // 👇️ clear timeout when the component unmounts
      clearTimeout(timeoutID);
    };
  }, []);